server/src/main/java/org/tctalent/server/service/db/UserService.java

```diff
@@ -80,12 +79,13 @@ public interface UserService {
     void updatePassword(UpdateUserPasswordRequest request);
     void updateUserPassword(long id, UpdateUserPasswordRequest request);
 
-
-    // Generates an email verification token and sends a verification email.
+    /**
+     * Generates an email verification token and sends a verification email.
+     */
     void sendVerifyEmailRequest(SendVerifyEmailRequest request);
-    // Checks the validity of the email verification token.
-    void checkEmailVerificationToken(CheckEmailVerificationTokenRequest request);
-    // Verifies the user's email using the verification token.
+    /**
+     * Checks the validity of the email verification token and verifies the user's email.
+     */
```

Comment:
This does not say what happens if the verification fails. It throws an Exception which should be documented.